Backend/src/main/java/Commands/UserCommands/UserAddBio.java

```diff
@@ -1,38 +1,19 @@
-package Commands.UserCommands;
-
-import Entities.HttpResponseTypes;
-import Interface.ConcreteCommand;
-import Models.Message;
-import Models.UserData;
-import com.arangodb.ArangoDB;
-import com.arangodb.entity.DocumentEntity;
-import com.arangodb.model.DocumentUpdateOptions;
-import org.json.JSONObject;
-
-public class UserAddBio extends ConcreteCommand {
-
-    @Override
-    protected HttpResponseTypes doCommand() {
-        ArangoDB arangoDB=ArangoInstance.getArangoDB();
-        UserData userData = ArangoInstance.getUserData(message.getUserID());
-        DocumentEntity res = null;
-        if (userData != null) {
-            userData.setBio(message.getBio());
-            res = arangoDB.db(ArangoInstance.getDbName()).collection("users").updateDocument(message.getUserID(), userData, new DocumentUpdateOptions().returnNew(true));
-        } else {
-               return HttpResponseTypes._404;
-        }
-        JSONObject response = new JSONObject();
-        JSONObject userDataJSON = new JSONObject(gson.toJson(res));
-        System.out.println(userDataJSON);
-        response.put("userData", userDataJSON);
-        responseJson = jsonParser.parse(response.toString());
-        System.out.println(response);
-        return HttpResponseTypes._200;
-    }
-
-    @Override
-    public void setMessage(Message message) {
-
-    }
-}
+package Commands.UserCommands;
+
+import Interface.ConcreteCommand;
+//import Models.User;
+import Models.Message;
+import com.arangodb.entity.DocumentEntity;
+import com.google.gson.Gson;
+import com.google.gson.JsonParser;
+import org.json.JSONObject;
+import Database.ArangoInstance;
+import Models.UserData;
+public class UserAddBio extends ConcreteCommand {
```

Comment:
Rename this class to UpdateUserData and remove the existing UpdateUserData.java

Reply:
I left both on purpose so they could be examples to either use custom logic or use the default methods. Okay I will remove them